README.md

```diff
@@ -121,6 +121,9 @@ By default, you are asked to overwrite or write to a separate file (`_ide_helper
 You can write the comments directly to your Model file, using the `--write (-W)` option, or
 force to not write with `--nowrite (-N)`.
 
+Alternatively using the `--write-mixin (-M)` option will only add a mixin tag to your Model file,
+writing the rest in (`_ide_helper_models.php`).
+
```

Comment:
I think the README fails to "sell" the feature so people understand what it does: the name different from the model, avoiding the IDE duplicate annoyance.

I fail to come up with a good description, but IMHO this super-pro should be written here 😄